data-reporting-service/src/main/resources/db/odse/019-fn_sql_server_agent_status_override.sql

```diff
@@ -0,0 +1,17 @@
+-- This function needs to be created in Azure since the Sql Server agent is not exposed
+-- Debezium deployment in Azure AKS need to override this function in the helm chart
+CREATE FUNCTION dbo.IsSqlAgentRunning() RETURNS BIT AS
```

Comment:
Does this need an ALTER statement as well? Or just create works?

Reply:
Thanks for catching this. Updated.